src/containers/AddProfile/index.js

```diff
@@ -61,6 +65,21 @@ const AddProfile = ({
   const [confirmModal, setConfirmModal] = useState(false);
   const [isFormDirty, setIsFormDirty] = useState(false);
 
+  useEffect(() => {
+    form.setFieldsValue({
+      profileType: history?.location?.cloneDetails?.profileType,
+      name: history?.location?.cloneDetails?.name
+        ? `${history?.location?.cloneDetails?.name} - CLONE`
+        : '',
+    });
+    setType(history?.location?.cloneDetails?.profileType);
+    setName(
+      history?.location?.cloneDetails?.name
+        ? `${history?.location?.cloneDetails?.name} - CLONE`
+        : ''
+    );
+  }, [history]);
+
```

Comment:
remove references to Clone and add a prop called `initialValues` to set the initial values of the form